tests/integration/test_vid_club.py

```diff
@@ -49,12 +52,46 @@ def test_build_query_wrong_source():
         )
 
 
+@pytest.mark.intervals
+def test_intervals_split():
+    from_date = "2022-01-01"
+    to_date = "2022-01-19"
+    days_interval = 5
+    expected_starts = ["2022-01-01", "2022-01-06", "2022-01-11", "2022-01-16"]
+    expected_ends = ["2022-01-06", "2022-01-11", "2022-01-16", "2022-01-19"]
+    starts, ends = vc.intervals(
+        from_date=from_date, to_date=to_date, days_interval=days_interval
+    )
+
+    assert starts == expected_starts
+    assert ends == expected_ends
+
```

Comment:
Please add docstring to test function that will shortly summarize what they are testing